README.md

```diff
@@ -7,9 +7,9 @@ We help companies build, run, deploy and scale software and infrastructure by em
 ---
 
 ## Description
-<$chart-name> Helm chart.
+bitbucket-runner-autoscaler Helm chart.
```

Comment:
Add link to https://bitbucket.org/bitbucketpipelines/runners-autoscaler/src/master/